go/vt/vtgate/evalengine/abs_test.go

```diff
@@ -0,0 +1,76 @@
+package evalengine
+
+import (
+	"math"
+	"testing"
+)
+
+func Test_buildinABS_call(t *testing.T) {
```

Comment:
Thanks for writing tests for this feature! However, we're trying to test the `evalengine` more reliably against MySQL itself: we write integration tests that verify the behavior of the function against a live MySQL instance. You can see an example here: https://github.com/vitessio/vitess/blob/main/go/vt/vtgate/evalengine/integration/comparison_test.go#L524-L551

Reply:
Do I need to submit more test cases？